Dockerfile

```diff
@@ -109,7 +110,7 @@ RUN for file in $(find /usr/share/postgresql -name 'postgresql.conf.sample'); do
     done
 
 # timescaledb-tune, as well as timescaledb-parallel-copy
-RUN echo "deb https://packagecloud.io/timescale/timescaledb/debian/ $(lsb_release -s -c) main" > /etc/apt/sources.list.d/timescaledb.list
+RUN echo "deb https://packagecloud.io/timescale/timescaledb/debian/ buster main" > /etc/apt/sources.list.d/timescaledb.list
```

Comment:
Then I think it's better to make a comment somewhere about it
```suggestion
# temporary using buster instead of $(lsb_release -s -c)
RUN echo "deb https://packagecloud.io/timescale/timescaledb/debian/ buster main" > /etc/apt/sources.list.d/timescaledb.list
```

Or even check somewhere that `curl -o /dev/null -s -w "%{http_code}\n" https://packagecloud.io/timescale/timescaledb/debian/` will return 404 (well, it's an overkill IMO)